h2o-persist-gcs/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     compile 'com.google.cloud:google-cloud-storage:1.19.0'
     testCompile "junit:junit:${junitVersion}"
     testCompile project(path: ":h2o-core", configuration: "testArchives")
+    testCompile project(":h2o-core")
```

Comment:
You only need this because you marked h2o-core as compileOnly ;)

Reply:
I know. This is resolved in next commit.